### PR TITLE
Add French contributing bucket

### PR DESCRIPTION
**Pull request by minimalsm**
## Description
Add French (`fr`) contributing bucket

### CODE REVIEW

File: src/content/translations/fr/contributing/design-principles/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+Vous pouvez voir nos principes de conception en action [dans l'ensemble de notre site](/).
+
+## Donnez-nous votre avis {#give-feedback}
+
+**Partagez vos commentaires sur ce document !** L'un des principes que nous proposons est l'« **amélioration collaborative** », ce qui signifie que nous voulons que le site web soit le produit de nombreux contributeurs. Dans l'esprit de ce principe, nous voulons donc partager ces principes de conception avec la communauté Ethereum.
+
+Bien que ces principes soient axés sur le site web ethereum.org, nous espérons que beaucoup d'entre eux sont représentatifs des valeurs de l'écosystème Ethereum dans son ensemble (par exemple, vous pouvez voir l'influence des [principes du livre blanc Ethereum](https://github.com/ethereum/wiki/wiki/White-Paper#philosophy)). Peut-être voudrez-vous même incorporer certains d'entre eux dans votre propre projet !
+
+Faites-nous part de vos réflexions sur le [serveur Discord](https://discord.gg/CetY6Y4) ou en [créant un ticket](<(https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=Type%3A+Feature&template=feature_request.md&title=)>).
```
Comment (corwintines) [inline src/content/translations/fr/contributing/design-principles/index.md:94]: This looks a little wonky

Comment (corwintines) [reply src/content/translations/fr/contributing/design-principles/index.md:94]:
```suggestion
Faites-nous part de vos réflexions sur le [serveur Discord](https://discord.gg/CetY6Y4) ou en [créant un ticket](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=Type%3A+Feature&template=feature_request.md&title=).
```
I think this should fix this

Comment (minimalsm) [reply src/content/translations/fr/contributing/design-principles/index.md:94]: Nice catch. This was a problem in all languages 😩 fixed here in #7417

File: src/content/translations/fr/contributing/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- [@ethdotorg](https://twitter.com/ethdotorg)
+- [Serveur Discord](https://discord.gg/CetY6Y4)
+
+### Remarque sur le plagiat {#plagiarism}
+
+Utilisez uniquement votre travail original ou le contenu que vous avez l'autorisation d'utiliser lorsque vous contribuez à un contenu ou à un artefact sur ethereum.org. De nombreux projets au sein de l'écosystème Ethereum utilisent des licences open source qui permettent le libre partage d'informations. Toutefois, si vous ne trouvez pas ces informations, ne tentez pas de les ajouter à ethereum.org. Toute PR considérée comme du plagiat sera rejetée.
+
+## Nouveau dans l'open source ? {#new-to-open-source}
+
+Dans notre environnement de dépôt GitHub, nous avons une catégorie de tickets spécialement conçue pour les développeurs qui sont nouveaux dans l'open source. Ces tickets de difficulté moindre sont étiquetés « <a href = « https://github.com/ethereum/ethereum-org-website/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22">good first issue</a> » (bon choix pour un premier ticket).
```
Comment (minimalsm) [inline src/content/translations/fr/contributing/index.md:65]:
```suggestion
Dans notre environnement de dépôt GitHub, nous avons une catégorie de tickets spécialement conçue pour les développeurs qui sont nouveaux dans l'open source. Ces tickets de difficulté moindre sont étiquetés « [good first issue](https://github.com/ethereum/ethereum-org-website/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) » (bon choix pour un premier ticket).
```

**Review by corwintines — APPROVED** [review]
LGTM

**Pull request merged**